packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -427,14 +427,14 @@ export class OdspDocumentStorageService extends OdspDocumentStorageServiceBase {
 			if (!versionsResponse) {
 				throw new NonRetryableError(
 					"No response from /versions endpoint",
-					DriverErrorType.genericNetworkError,
+					DriverErrorTypes.genericNetworkError,
```

Comment:
I believe the uses of `DriverErrorTypes` here can be replaced with `OdspErrorTypes`, as the later should include all the flags of the former. This should be applicable to usages across this package, and similarly applicable for `routerlicious-driver`.

Reply:
Changed`DriverErrorTypes` to `OdspErrorTypes`.